### ISSUE TITLE
Add FY23 Fact Sheet

### ISSUE THREAD

**Issue by LoraBradford** [GSA/smartpay-website#439]
Upload FY23 Fact Sheet to the Program website

**Comment by CodyHinze** [comment]
**Here is a list of the accessibility issues I found in ths this PowerPoint slide deck:** 

- The four card images and the GSA logo are missing alt text. Those can be added by right-clicking on the images and choosing. "View Alt Text". These images are currently marked as "decorative," so they are not picked up by the accessibility test tool but are not decorative. They should include alt text such as we use on the SmartPay program home page: "Red charge card with the word Purchase and numbers 1234 5678 9012 3456 and the name John Smith, with a bird in the background"
- The Title on the second page is the Email at the bottom instead of the "Fiscal Year 2023" header.  This can be fixe by opening the Accessibility checker option in the "Review" ribbon and then choosing the Slide Title option and "Set Slide Title option.

The color contrast, and reading order was fine.

**I also found a couple additional layout issues:**
- The second page includes two copies of the GSA logo on top of each other, the duplicate should be removed.
- The card images are cropped differently and have different alignment and corner radiuses. These images should pre places again and we should remove the border and corner radiuses. 


**Open questions:**

Would you like a version with those changes made? 

Also, what is the final format for this going to be when it is published online? Are we planning on making this into a PDF?
